requirements/requirements.txt

```diff
@@ -66,6 +66,7 @@ django-compressor==2.1
 django-angular==0.7.16
 Pycco==0.5.1
 django-mptt==0.8.7
+django-cte==0.1.3
```

Comment:
[django-cte](https://github.com/dimagi/django-cte) should be reviewed as well, since it is part of the work that went into this PR. It works on Python 2 and 3, has its own test suite, and is published to PyPI. It is a stand-alone library so it can hopefully help with the effort to [add CTE support to Django](https://code.djangoproject.com/ticket/28919).